config/routes.rb

```diff
@@ -1,6 +1,16 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :entity
+  get 'entity/index'
+  get 'entity/show'
+  get 'entity/new'
+  post '/entity/new', to: "entity#create", as: 'entity_create'
+  devise_for :users
+  get 'splash/show'
+  root 'splash#show'
+  get 'categories/index'
+  get 'categories/new'
+  resources :categories
+  devise_scope :user do
+    get '/users/sign_out' => 'devise/sessions#destroy'
+  end
```

Comment:
- [ ] Great work creating the resources you need for this project. However, one of the best practices in Ruby on Rails is to add only necessary routes. Currently, you are creating routes for the `index` and `show` page of your `Category` and `Entity` models twice. To fix this, I would advise you to use the resources method for creating routes and then specify the ones you wish to have.

```ruby on rails
resources :categories, only: [:index, :show]
resources :entities, only: [:index, :show]
```
You can choose to do it this way or explicitly create each one of them using their HTTP verb or request method. 👍